Mon Sep 12 18:32:14 2022 - INFO  - Start environmental preparation.
Mon Sep 12 18:33:07 2022 - INFO  - End of environmental preparation!
Mon Sep 12 18:33:08 2022 - INFO  - Start testing...
Failed to restart plymouth-start.service: Unit plymouth-start.service not found.
Mon Sep 12 18:33:08 2022 - ERROR - plymouth-start.service restart failed
Mon Sep 12 18:33:09 2022 - ERROR - ../common/common_lib.sh line 34
Unit plymouth-start.service could not be found.
Mon Sep 12 18:33:16 2022 - ERROR - plymouth-start.service restart failed
Mon Sep 12 18:33:16 2022 - ERROR - ../common/common_lib.sh line 37
Failed to stop plymouth-start.service: Unit plymouth-start.service not loaded.
Mon Sep 12 18:33:17 2022 - ERROR - plymouth-start.service stop failed
Mon Sep 12 18:33:18 2022 - ERROR - ../common/common_lib.sh line 39
Unit plymouth-start.service could not be found.
Mon Sep 12 18:33:24 2022 - ERROR - plymouth-start.service stop failed
Mon Sep 12 18:33:25 2022 - ERROR - ../common/common_lib.sh line 42
Failed to start plymouth-start.service: Unit plymouth-start.service not found.
Mon Sep 12 18:33:26 2022 - ERROR - plymouth-start.service start failed
Mon Sep 12 18:33:26 2022 - ERROR - ../common/common_lib.sh line 44
Unit plymouth-start.service could not be found.
Mon Sep 12 18:33:33 2022 - ERROR - plymouth-start.service start failed
Mon Sep 12 18:33:34 2022 - ERROR - ../common/common_lib.sh line 47
Failed to get unit file state for plymouth-start.service: No such file or directory
Mon Sep 12 18:33:34 2022 - INFO  - Unit is indirect, ignoring.
Failed to start plymouth-start.service: Unit plymouth-start.service not found.
Mon Sep 12 18:33:35 2022 - ERROR - Job type reload is not applicable for unit plymouth-start.service
Mon Sep 12 18:33:36 2022 - ERROR - ../common/common_lib.sh line 91
Unit plymouth-start.service could not be found.
Unit plymouth-start.service could not be found.
Mon Sep 12 18:33:37 2022 - INFO  - Finish test!
Mon Sep 12 18:33:37 2022 - ERROR - failed to execute the case.
Mon Sep 12 18:33:38 2022 - INFO  - start environment cleanup.
Failed to stop plymouth-start.service: Unit plymouth-start.service not loaded.
oe_test_service_plymouth-start.sh: line 38: plymouth: command not found
Traceback (most recent call last):
  File "/root/GitRepo/mugen-riscv/libs/locallibs/rpm_manage.py", line 173, in <module>
    exitcode = rpm_remove(args.node, args.pkgs, args.tempfile)
  File "/root/GitRepo/mugen-riscv/libs/locallibs/rpm_manage.py", line 144, in rpm_remove
    with open(tmpfile, "r") as f:
OSError: [Errno 36] File name too long: "Last metadata expiration check: 0:11:00 ago on Mon 12 Sep 2022 06:22:05 PM CST.\nError: \n Problem: conflicting requests\n  - nothing provides system-logos needed by plymouth-0.9.5-4.oe2209.riscv64\n(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)"
Mon Sep 12 18:33:41 2022 - INFO  - Finish environment cleanup!
